gateway-server/src/main/java/org/apache/knox/gateway/util/KnoxCLI.java

```diff
@@ -701,31 +717,34 @@ protected RemoteConfigurationRegistryClientService getRemoteConfigRegistryClient
 
  private class AliasListCommand extends Command {
 
-  public static final String USAGE = "list-alias [--cluster clustername]";
+  public static final String USAGE = "list-alias [--cluster cluster1:clusterN]";
   public static final String DESC = "The list-alias command lists all of the aliases\n" +
-                                    "for the given hadoop --cluster. The default\n" +
+                                    "for the given hadoop --cluster(s). The default\n" +
                                     "--cluster being the gateway itself.";
 
    @Override
    public void execute() throws Exception {
      AliasService as = getAliasService();
-      KeystoreService keystoreService = getKeystoreService();
+     KeystoreService keystoreService = getKeystoreService();
 
      if (cluster == null) {
        cluster = "__gateway";
      }
-      boolean credentialStoreForClusterAvailable =
-          keystoreService.isCredentialStoreForClusterAvailable(cluster);
-      if (credentialStoreForClusterAvailable) {
-        out.println("Listing aliases for: " + cluster);
-        List<String> aliases = as.getAliasesForCluster(cluster);
-        for (String alias : aliases) {
-          out.println(alias);
-        }
-        out.println("\n" + aliases.size() + " items.");
-      } else {
-        out.println("Invalid cluster name provided: " + cluster);
-      }
+     String[] clusters = cluster.split(":");
```

Comment:
1. `:` should be a constant e.g. CLUSTER_STRING_SEPARATOR
2. I think `,` would be more obvious, no?
3. What if the cluster had `:` it should not but we had a case where alias name had special characters.

Reply:
Yes, it makes more sense to use `,`. I also added it as a constant.